### PR TITLE
Topic recognition s4608425

### PR DESCRIPTION
**Pull request by tusharvinayaraj**
Hi Shakes,

This is my submission for task3 : implementing unet using ISIC dataset.

I implemented the model in the python file, the jupyter notebook has code for running the same. Necessary details are listed in readme file.

I have used tensorflow for implementing the model as well as for the calculation of dice similarity. Dice value got is 0.62 approx.

Any suggestions will be highly appreciated.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific
- Code valid for Easy Difficulty
- Results are okay, DSC below 0.7

Other Feedback
- Please follow python file  and folder naming convention e.g. `lorem_ipsum_123_abc.py`
- Your images appear as links in the MD. You should be able to show them in text

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Tensorflow has functions for image resizing and more so there’s no need to also use PIL in your project. 

Paths such as "C:/Users/s4608425/Downloads/ISIC2018_Task1_Training_GroundTruth_x2/*.png" won’t work for the user and should either be set dynamical or it needs to be made clear to the user that they need to change these paths

Comments such as 
   #compile model
   model.compile(metrics=['accuracy'], loss="binary_crossentropy",optimizer='adam')
Add nothing to the readability of the code and should be avoided. Overall you needed a few more meaningful comments

Code in files such as driverscipt.ipynb should be in a main function in a python file.

File: recognition/s4608425/MODEL_imp.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import numpy as np
```
Comment (shakes76) [inline recognition/s4608425/MODEL_imp.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme Limited
Module NOT present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
